[INFO   ] Logger: Record log in /storage/emulated/0/UI/.kivy/logs/kivy_22-02-02_62.txt
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.9.7 (default, Oct  6 2021, 01:34:26) 
[GCC 11.1.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] KivyMD: 0.104.2, git-bc7d1f5, 2021-06-06 (installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivymd/__init__.py")
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 v1.r26p0-01eac0.035aefc9426b37ec2259de8e3dc4e984'>
[INFO   ] GL: OpenGL vendor <b'ARM'>
[INFO   ] GL: OpenGL renderer <b'Mali-G52 MC2'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <8192>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] GL: NPOT texture support is available
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 242, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/storage/emulated/0/UI/nav.kv", line 14, in <module>
[WARNING] stderr:     pos: (self.ids[ico].center[0]-12, self.ids[ico].center[1]-54)
[WARNING] stderr: NameError: name 'ico' is not defined
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 695, in _apply_rule
[WARNING] stderr:     value, bound = create_handler(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 245, in create_handler
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "/storage/emulated/0/UI/nav.kv", line 14:
[WARNING] stderr: ...
[WARNING] stderr:      12:                font_size: "50sp"
[WARNING] stderr:      13:                color: 1,0,1,1
[WARNING] stderr: >>   14:                pos: (self.ids[ico].center[0]-12, self.ids[ico].center[1]-54)
[WARNING] stderr:      15:                size_hint: None, None
[WARNING] stderr:      16:            MDBoxLayout:
[WARNING] stderr: ...
[WARNING] stderr: NameError: name 'ico' is not defined
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 242, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/storage/emulated/0/UI/nav.kv", line 14, in <module>
[WARNING] stderr:     pos: (self.ids[ico].center[0]-12, self.ids[ico].center[1]-54)
[WARNING] stderr: 
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/UI/main.py", line 17, in <module>
[WARNING] stderr:     MainApp().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 949, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 919, in _run_prepare
[WARNING] stderr:     root = self.build()
[WARNING] stderr:   File "/storage/emulated/0/UI/main.py", line 15, in build
[WARNING] stderr:     return NavScreen()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/relativelayout.py", line 274, in __init__
[WARNING] stderr:     super(RelativeLayout, self).__init__(**kw)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/floatlayout.py", line 65, in __init__
[WARNING] stderr:     super(FloatLayout, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/layout.py", line 76, in __init__
[WARNING] stderr:     super(Layout, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivymd/uix/behaviors/backgroundcolor_behavior.py", line 220, in __init__
[WARNING] stderr:     super().__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivymd/uix/behaviors/backgroundcolor_behavior.py", line 178, in __init__
[WARNING] stderr:     super().__init__(**kwarg)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivymd/uix/behaviors/elevation.py", line 847, in __init__
[WARNING] stderr:     super().__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/widget.py", line 359, in __init__
[WARNING] stderr:     self.apply_class_lang_rules(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/widget.py", line 463, in apply_class_lang_rules
[WARNING] stderr:     Builder.apply(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 541, in apply
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 710, in _apply_rule
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "/storage/emulated/0/UI/nav.kv", line 14:
[WARNING] stderr: ...
[WARNING] stderr:      12:                font_size: "50sp"
[WARNING] stderr:      13:                color: 1,0,1,1
[WARNING] stderr: >>   14:                pos: (self.ids[ico].center[0]-12, self.ids[ico].center[1]-54)
[WARNING] stderr:      15:                size_hint: None, None
[WARNING] stderr:      16:            MDBoxLayout:
[WARNING] stderr: ...
[WARNING] stderr: BuilderException: Parser: File "/storage/emulated/0/UI/nav.kv", line 14:
[WARNING] stderr: ...
[WARNING] stderr:      12:                font_size: "50sp"
[WARNING] stderr:      13:                color: 1,0,1,1
[WARNING] stderr: >>   14:                pos: (self.ids[ico].center[0]-12, self.ids[ico].center[1]-54)
[WARNING] stderr:      15:                size_hint: None, None
[WARNING] stderr:      16:            MDBoxLayout:
[WARNING] stderr: ...
[WARNING] stderr: NameError: name 'ico' is not defined
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 242, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/storage/emulated/0/UI/nav.kv", line 14, in <module>
[WARNING] stderr:     pos: (self.ids[ico].center[0]-12, self.ids[ico].center[1]-54)
[WARNING] stderr: 
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 695, in _apply_rule
[WARNING] stderr:     value, bound = create_handler(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 245, in create_handler
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: 
